.changes/unreleased/🎉 Feature-20230116-184305.yaml

```diff
@@ -0,0 +1,8 @@
+kind: "\U0001F389 Feature"
+body: Add sort, sorted-by flags to policy search command.
+time: 2023-01-16T18:43:05.47579277+02:00
+custom:
+  azure-boards-workitemid-fixed: '481928'
```

Comment:
The fixed value means it will automatically change this work item state in the backlog.

Just wanted you to be aware it will close out the PBI or the task.